src/Compilers/CSharp/Portable/Symbols/LocalDeclarationKind.cs

```diff
@@ -49,5 +49,16 @@ internal enum LocalDeclarationKind : byte
         /// The variable that captures the result of a pattern matching operation like "i" in "expr is int i"
         /// </summary>
         PatternVariable,
+
+        /// <summary>
+        /// User variable declared by a declaration expression in the left-hand-side of a deconstruction assignment.
+        /// </summary>
+        Deconstruction,
+
+        /// <summary>
+        /// User variable declared by a declaration expression that is not a deconstruction. This occurs for an
+        /// out variable declaration, and as a result of error recovery in incorrect code.
+        /// </summary>
+        DeclarationExpression,
```

Comment:
I am not suggesting to merge anything. I am asking a question why you've chosen to treat variables declared by the same syntax differently, contrary to the proposed design and your previous response. I am simply observing that we are using different "precision" for syntax for different kinds of variables. This approach is inconsistent, we are not here, and not there. I believe that we should have consistent approach. It could be less accurate or more accurate, but with the same degree of accuracy for all.